package.json

```diff
@@ -519,7 +534,9 @@
         "r.helpPanel.rpath": {
           "type": "string",
           "default": "",
-          "markdownDescription": "Path to an R executable. Must be \"vanilla\" R, not radian etc.! Will be read from registry or path if not set."
+          "markdownDescription": "Path to an R executable. Must be \"vanilla\" R, not radian etc.! Will be read from registry or path if not set.",
+          "markdownDeprecationMessage": "Will be deprecated. Use `#r.rpath.windows#`, `#r.rpath.mac#`, or `#r.rpath.linux#` instead.",
```

Comment:
Looks like the deprecation message is not shown in either settings visual editor or json editor:

This entry no longer appears in the completion of settings:

<img width="892" alt="image" src="https://user-images.githubusercontent.com/4662568/100164796-982dc880-2ef3-11eb-936e-cd6e58e37a5a.png">

But if I use this settings, then I don't see any deprecation warnings in either the description or the diagnostics:

<img width="527" alt="image" src="https://user-images.githubusercontent.com/4662568/100164838-b267a680-2ef3-11eb-95c7-126bbe47c345.png">

Reply:
That's weird... Have you tried packaging+installing the modified code and checking in a new vscode instance? Sometimes that changes to package.json don't take effect when debugging the extension normally

Reply:
I build all three extensions and installed the three vsix files.

Reply:
Do the deprecation warnings work in any of the other two extensions?

Just for info, this is how the warnings are supposed to look:

![deprecation1](https://user-images.githubusercontent.com/53863351/100214276-fcf71c00-2f0f-11eb-9eae-543fe5d2861a.PNG)

![deprecation2](https://user-images.githubusercontent.com/53863351/100214284-01bbd000-2f10-11eb-942c-74552e6ef1d6.PNG)

Reply:
Any updates on this? I checked the settings UI on a linux VM and both types of deprecation messages work there

Reply:
In the settings viewer, the deprecation message looks nice:

<img width="681" alt="image" src="https://user-images.githubusercontent.com/4662568/100516341-38b70d80-31be-11eb-98c5-a34e94868fba.png">

But in the `settings.json` editor, I don't see the deprecation message:

<img width="566" alt="image" src="https://user-images.githubusercontent.com/4662568/100516372-6734e880-31be-11eb-8c01-f070597b8ed9.png">

Reply:
Thanks for the update. Since I cannot reproduce the issue, I probably won't be able to fix this. I'd suggest we just add e.g. `DEPRECATED! ...` to the front of the normal description to make sure people with this error notice the change (hoping that the proper deprecation message works for most users)

(And as we still honour the old setting in this PR, it's not a breaking change even if users don't notice)

Reply:
Sounds good to me.